package.json

```diff
@@ -0,0 +1,189 @@
+{
+  "name": "@chainsafe/libp2p-yamux",
+  "version": "0.1.0",
+  "description": "Yamux stream multiplexer for libp2p",
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/ChainSafe/js-libp2p-yamux#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/js-libp2p-yamux.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ChainSafe/js-libp2p-yamux/issues"
+  },
+  "keywords": [
+    "IPFS",
+    "libp2p",
+    "stream",
+    "multiplexer",
+    "muxer"
+  ],
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
+  "type": "module",
+  "types": "./dist/src/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "*": [
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
+      ],
+      "src/*": [
+        "*",
+        "dist/*",
+        "dist/src/*",
+        "dist/src/*/index"
+      ]
+    }
+  },
+  "files": [
+    "src",
+    "dist/src",
+    "!dist/test",
+    "!**/*.tsbuildinfo"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/src/index.js"
+    },
+    "./config": {
+      "import": "./dist/src/config.js"
+    },
+    "./stream": {
+      "import": "./dist/src/stream.js"
+    }
+  },
+  "eslintConfig": {
+    "extends": "ipfs",
+    "parserOptions": {
+      "sourceType": "module"
+    },
+    "ignorePatterns": [
+      "src/*.d.ts"
+    ]
+  },
+  "release": {
```

Comment:
don't need this with our pipeline. I don't think it works since you didn't add that dep anyways